docs/design/README.md

```diff
@@ -8,6 +8,9 @@ Generally, design docs are on Google docs:
 * [Sync Resource](https://docs.google.com/document/d/1ZbXaEh7v_HcgrRu7N-kEucU2pXyq1Rt7cSmmFOBN2q8/edit)
 * [External Data Response Cache](https://docs.google.com/document/d/1nl6iWMmh9CRT0A6cr-Nkfofv9wp210Bkf6ilLq9LLBo/edit)
 * [Gator SyncSet Support](https://docs.google.com/document/d/1mBvy6Y7TDyZswdYlFBHB3Mi2irLVqga_7ZoZvfPIU0A/edit)
+* [Scoping enforecment actions](https://docs.google.com/document/d/1qEzDVcmwPysb3ejvTYKPkHJm3Db3Yr7zShwmgFOsR_k/edit#heading=h.6ewsycnqnfl)
+* [Configuring VAP as enforcement point](https://docs.google.com/document/d/1u55cMZ4smRAuEZf4BYsSyZOCoOy33Lsgp3xuwDm0l6I/edit#heading=h.9oaxevykg8w6)
```

Comment:
This doc doesn't currently capture the decision from the Slack discussion. Instead of keeping the doc, how about add the decision from the discussion to the `Gatekeeper-to-VAP User Flow` doc specifically add a new section under https://docs.google.com/document/d/1UELB0dmt10K5jBwETUnSKKQN7qeivOjvi-AvzFTOsHw/edit#heading=h.557kq3533dz3 ?

Reply:
Can you also add the `Gatekeeper-to-VAP User Flow` doc https://docs.google.com/document/d/1UELB0dmt10K5jBwETUnSKKQN7qeivOjvi-AvzFTOsHw/edit#heading=h.557kq3533dz3

Reply:
I linked to the `User facing-changes` section of the doc https://docs.google.com/document/d/1UELB0dmt10K5jBwETUnSKKQN7qeivOjvi-AvzFTOsHw/edit#heading=h.557kq3533dz3 so yes, please add a new sub section under that section to capture the decision. thanks!